infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/LayoutSearchService.kt

```diff
@@ -24,6 +24,7 @@ constructor(
         limitPerResultType: Int,
         locationTrackSearchScope: IntId<LocationTrack>?,
         searchedAssetTypes: List<TrackLayoutSearchedAssetType>,
+        includeDeleted: Boolean,
```

Comment:
Näyttää siltä että täällä tiedostossa olisi muuttunut paljon juttuja, mutta oikeasti kaikki muutokset ovat vaan `includeDeleted`:in passailua ympäriinsä

Reply:
Joo, tuollainen voisi olla ihan hyvä malli. Muutan nämä sen kaltaisiksi